File: pset6/mario.py
import cs50

counter = 0
counter2 = -1
rownumber = 1
height = 0
while True:
    print("How high is the pyramid?")
    height = cs50.get_int()
    if height < 24 and height > 0:
        break
for rownumber in range(height+1):
    for counter in range(height-rownumber):
        print(" ",end="")
        counter=counter+1
    for counter2 in range(rownumber):
        print("#",end="")
        counter2=counter2+1
    print()
    rownumber=rownumber + 1

File: mario.c
#include <stdio.h>
#include <cs50.h>

int main (void)
{

    int counter = 0;
    int height = 0;
    do
    {
        printf("What is the height of the pyramid?\n");
        height = get_int();
    }
    while (height > 23 || height < 0);

    for (int row = 1; row <= height; row++)
    {
        for (counter = 0; counter < height - row; counter++)
            printf(" ");
        for(counter = -1;counter<row;counter++)
            printf("#");
        
            printf("\n");
    }

}